ngraph/frontend/ir/include/ir_frontend/model.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <frontend_manager/frontend_manager.hpp>
+#include <inference_engine.hpp>
```

Comment:
it looks very ugly that we use IE API in ngraph frontends. 
Can we use forward declaration at least?

Reply:
1. We can remove `InferenceEngine::Blob::CPtr` usage and replace it with `ngraph::op::Constant`
2. For extensions - we can directly pass opsets instead of `IExtensionPtr`

Reply:
Implemented: https://github.com/openvinotoolkit/openvino/pull/7249